src/cli/main.cc

```diff
@@ -109,7 +109,6 @@ static void InitGoogleLog(const Config *config) {
   } else {
     FLAGS_log_dir = config->log_dir + "/";
     if (config->log_retention_days != -1) {
-      google::EnableLogCleaner(std::chrono::hours(24) * config->log_retention_days);
```

Comment:
We don't need a thin line?

Reply:
Yes, I forget to revert this change. This line throws an error on my macOS when building. I workaround this by remove it temporially.